src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -940,6 +940,12 @@ void Graph::Allocate(const std::vector<size_t>& syncNodesInds) {
                 one_of(edge->getChild()->getType(), Type::Output, Type::MemoryOutput) &&
                 edge->inPlace(Edge::LOOK_DOWN)) {
                 edge->getChild()->resolveInPlaceEdges(Edge::LOOK_DOWN);
+            } else if (one_of(edge->getParent()->getType(), Type::MemoryInput)) {
+                auto memInp = std::dynamic_pointer_cast<node::MemoryInput>(edge->getParent());
+                if (memInp && memInp->haveSubgraph()) {
+                    // Since the ReadValueWithSubgraph is middle node, just add this branch in order to assign memBlock.
+                    edge->getParent()->resolveInPlaceEdges(Edge::LOOK_UP);
```

Comment:
Please check https://github.com/openvinotoolkit/openvino/pull/26130#discussion_r1800861084

Reply:
[Commented](https://github.com/openvinotoolkit/openvino/pull/26130#discussion_r1809779154)

Removed, but current implementation will import `MemoryOutputSingleStub`, it still assign memory, but does not inference. `MemoryOutputSingleStub`'s type still is `MemoryOutput`, [ProxyMemoryBlockPtr memBlock](https://github.com/openvinotoolkit/openvino/pull/26130/files#diff-b3d6b84c29252e15ea9aa04ff740d446e389be996bb58dd4638a98233dbdd71eR106) will still be assiginned.